apps/src/applab/ConnectedStudioAppWrapper.jsx

```diff
@@ -0,0 +1,19 @@
+/** @file Redux-connected version of StudioAppWrapper, used by App Lab. */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var connect = require('react-redux').connect;
+var StudioAppWrapper = require('../templates/StudioAppWrapper.jsx');
+
+module.exports = connect(
+  (state) => ({
+    assetUrl: state.assetUrl,
+    isEmbedView: state.isEmbedView,
+    isShareView: state.isShareView
+  })
+)(StudioAppWrapper);
```

Comment:
I don't love the idea of having two different components, one of which is connected and one of which isnt. Do we expect this to just be necessary for transitional purposes, or a more long term approach?